templates/graphdb-node.yaml

```diff
@@ -120,6 +132,7 @@ spec:
               mountPath: /opt/graphdb/home
             - name: graphdb-license
               mountPath: /tmp/license/
+          securityContext: {{ $.Values.graphdb.node.containerSecurityContext | toYaml | nindent 12 }}
```

Comment:
First time seeing `containerSecurityContext` as value. Try to stick with what the rest of the community charts use: `securityContext` and `podSecurityContext`, `helm create` does exactly this.

Reply:
Both the mysql, postgresql and neo4j have containerSecurityContext

Reply:
Touché, you can still consider it tho.

Reply:
Changed it to podSecurityContext and securityContext